### DISCUSSION TITLE
/cmi aliaseditor help

### DISCUSSION THREAD

**Discussion by UreshiiiDesu** [Zrips/CMI#5367] [Q&A]
When i make an alias for sethome, i write {/cmi aliaseditor new sethome-cmi sethome}. This only allows for players to set one home as it doesn't allow for custom names, so it just recreates the home home every time. Anybody have any idea how to fix it? Also sorry if it was already posted, I tried looking but I couldn't see ^^'

**Comment by dkalaxdk** [comment]
That is because you are missing the $1- variable.
However, go into alias.yml and just set sethome to true ;)

**Reply by mrfloris** [reply]
This is correct, any `/cmi <command>` can be used as `/<command>`, more info about that here:
https://github.com/mrfdev/CMI/blob/master/Resources/FAQ/cmi-commands.md

Using `cmi checkperm sethome` you can find home setting related permissions:
<img width="845" alt="Screenshot 2021-09-08 at 09 43 19" src="https://user-images.githubusercontent.com/28841349/132467636-ad0c9d1f-845e-412d-bb65-5ef4e5faae71.png">

`cmi.command.sethome.[anypositivenumber] - Allows to have defined amount of homes`

This is the one you want, for example to limit the default players to 1 home, but give vip 2 homes and staff 5, in luckperms you can do this:
```
lp group default permission set cmi.command.sethome.1 true
lp group vip permission set cmi.command.sethome.2 true
lp group staff permission set cmi.command.sethome.5 true
```
In `Alias.yml` you can set `/sethome` to true, and remove your own custom alias, .. then they can use a custom name, and everything behaves as expected.
